### DISCUSSION TITLE
AppApi autocof

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#3489] [Questions] [answered]
Not sure if this should be reported here or in the appapi repo, but https://cloud-py-api.github.io/app_api/DeployConfigurations.html#nextcloud-in-docker-aio-all-in-one says the deamon config is created automatically, but for me this didn't happend and the entrypoint.sh/start.sh doesn't configure appapi

**Comment by Zoey2936** [comment]
![Screenshot_20231005_185150_Samsung Internet](https://github.com/nextcloud/all-in-one/assets/75573284/2ada557c-ab16-4e00-988c-2e1bfe7115fb)
A
![Screenshot_20231005_185031_Samsung Internet](https://github.com/nextcloud/all-in-one/assets/75573284/d098791e-d8d3-489c-b79f-0443b9445d92)
Lsi this error with this conf

**Comment by szaimen** [comment]
Did you install the docker socket proxy container?

**Reply by Zoey2936** [reply]
yes

**Reply by Zoey2936** [reply]
![Screenshot_20231005_191114_Samsung Internet](https://github.com/nextcloud/all-in-one/assets/75573284/6a1f98c2-dd04-4f6c-bcde-0058be301b2b)

**Comment by szaimen** [comment, marked as answer]
I guess best to report this to the app then as it worked on my test instance...

**Reply by Zoey2936** [reply]
Can you please send me the config app api created for your?

**Reply by szaimen** [reply]
It was the same but with my own doamin so not sure why it reports errors when it tries to test it

**Reply by Zoey2936** [reply]
Is it possible to transfer this discussion? Or do I need to create a new?

**Reply by szaimen** [reply]
I fear you need to create a new one since the other repo is in a different org
